internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/ClassNameSpecification.scala

```diff
@@ -197,6 +198,7 @@ class ClassNameSpecification extends UnitSpec {
   }
 
   it should "not create binary names for local classes" in {
+    pending
```

Comment:
This test fails on 2.13.0-M5.

Reply:
Can you open a ticket about it?

Reply:
Here you go - https://github.com/sbt/zinc/issues/613